content/docs/command-reference/plots/diff.md

```diff
@@ -6,76 +6,77 @@ plotting them in a single image.
 ## Synopsis
 
 ```usage
-usage: dvc plots diff [-h] [-q | -v] [-t [TEMPLATE]] [-d [DATAFILE]] [-f FILE]
-                     [-s SELECT] [-x X] [-y Y] [--stdout] [--no-csv-header]
-                     [--no-html] [--title TITLE] [--xlab XLAB] [--ylab YLAB]
-                     [revisions [revisions ...]]
+usage: dvc plots diff [-h] [-q | -v] [-t <name_or_path>]
+                      [--targets [<path> [<path> ...]]] [-o <path>]
+                      [-x <field>] [-y <field>] [--no-csv-header]
+                      [--show-vega] [--title <text>]
+                      [--x-label <text>] [--y-label <text>]
+                      [revisions [revisions ...]]
 
 positional arguments:
-  revisions             Git commits to plot from
+  revisions             Git commits to plot from/to
 ```
 
 ## Description
 
-This command visualize difference between metrics among experiments in the
-repository history. Requires that Git is being used to version the metrics
-files.
+This command is a way to visualize the difference between metrics among
+experiments in the <abbr>repository</abbr> history. The target metric files
+(required) should specified with the `--targets` (`-t`) option.
 
-The metrics file needs to be specified through `-d`/`--datafile` option. Also, a
-plot can be customized with
-[plot templates](/doc/command-reference/plots#plot-templates) using the
-`--template` option. To learn more about the file formats and templates please
-see `dvc plots`.
+The `targets` should be <abbd>outputs</abbr> of one of the
```

Comment:
we don't have `targets` anymore?

Reply:
okay, I see that we have ... let's just change it to `--targets` everywhere

Reply:
Not sure what you mean. plots diff has --targets

Reply:
Oh, I guess it's not required anymore...

Reply:
the whole paragraph is not needed that much - try to make it part of the previous sentence. The value here is the reference to the --plots? just make it in the `( ... )`

Reply:
One min... All the options were rearranged again in the latest core version I have. Updating ⏳

Reply:
> let's just change it to --targets everywhere

This part is done.

Reply:
> the whole paragraph is not needed that much...

OK, I moved the just of it into the **Options** section for this cmd ref. Note that a similar paragraph is repeated in show and modify, where it makes more sense (esp. in modify).